<ui version="4.0" >
 <class>FindDialog</class>
 <widget class="QWidget" name="FindDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >
    <normaloff>:/resources/images/poshuku.png</normaloff>:/resources/images/poshuku.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Find what:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Pattern_" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="SearchBackwards_" >
       <property name="text" >
        <string>Search backwards</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="MatchCase_" >
       <property name="text" >
        <string>Match case</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="WrapAround_" >
       <property name="text" >
        <string>Continue from the beginning</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="FindButton_" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseButton_" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>CloseButton_</sender>
   <signal>released()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>365</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel" >
     <x>359</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
